jobs/javascript/http/job-scheduler/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "order-processor",
```

Comment:
`order-processor` not correct here either - this should be the name of the app-id, so `job-scheduler`